cypress/support/commands.js

```diff
@@ -23,19 +24,77 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-Cypress.Commands.add('mockWindowChrome', () => {
-    cy.window().then(
-        // one of the fec dependencies talks to window.insights.chrome
-        (window) =>
-            (window.insights = {
-                chrome: {
-                    getUserPermissions: () => ['inventory:*:*'], // enable all read/write features
-                    auth: {
-                        getUser: () => {
-                            return Promise.resolve({});
-                        }
-                    }
-                }
-            })
-    );
-});
\ No newline at end of file
+
+import React from 'react';
+import FlagProvider from '@unleash/proxy-client-react';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { mount } from '@cypress/react18';
+import { RBACProvider } from '@redhat-cloud-services/frontend-components/RBACProvider';
+import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities';
+
+Cypress.Commands.add('mountWithContext', (Component, options = {}, props) => {
+  const { path, routerProps = { initialEntries: ['/'] } } = options;
+
+  return mount(
+    <FlagProvider
+      config={{
+        url: 'http://localhost:8002/feature_flags',
+        clientKey: 'abc',
+        appName: 'abc',
+      }}
+    >
+      <Provider store={getRegistry().getStore()}>
+        <MemoryRouter {...routerProps}>
+          <RBACProvider appName="inventory" checkResourceDefinitions>
```

Comment:
Some components can check for permissions (`usePermissions` or `usePermissionsWithContext`)